components/FAQList.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react'
```

Comment:
uberniting: I ususally prefer, when using camelCase, to not capitalise acronims. Here `FaqList`, feel free to ignore. See https://stackoverflow.com/questions/15526107/acronyms-in-camelcase